orchagent/main.cpp

```diff
@@ -450,6 +451,12 @@ int main(int argc, char **argv)
                 vrf = optarg;
             }
             break;
+        case 'I':
+            if (optarg)
+            {
+                g_heart_beat_interval = atoi(optarg);
```

Comment:
>g_heart_beat_interval [](http://example.com/codeflow?start=16&length=21)

you may init g_heart_beat_interval before the while loop, instead of init global static var which the sequence is not well defined.

Reply:
Fixed